common/bytes_test.go

```diff
@@ -83,16 +83,45 @@ func (s *BytesSuite) TestCopyBytes(c *checker.C) {
 }
 
 func (s *BytesSuite) TestIsHex(c *checker.C) {
-	data1 := "a9e67e"
-	exp1 := false
-	res1 := IsHex(data1)
-	c.Assert(res1, checker.DeepEquals, exp1)
-
-	data2 := "0xa9e67e00"
-	exp2 := true
-	res2 := IsHex(data2)
-	c.Assert(res2, checker.DeepEquals, exp2)
+	tests := []struct {
+		data string
+		hex  bool
+	}{
+		// Ensure minimum length requirements pass
+		{"", false},
+		{"0", false},
+		{"00", false},
+		{"0x", false},
```

Comment:
That was the logic of the code previously. I didn't want to relax any checks performed.